project/src/main/java/com/google/sps/servlets/BlobstoreUrlServlet.java

```diff
@@ -36,7 +36,8 @@ public BlobstoreUrlServlet() {
   //Creates an upload url for an image using a cloud storage bucket
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String bucketName = "cloudberry-step-2020-test-bucket";
+    // String bucketName = "cloudberry-step-2020-test-bucket"; // Use twhen deploying to cloudberry.
+    String bucketName = "neelgandhi-step-2020-test-bucket"; // Use when deploying to neelgandhi.
```

Comment:
It's best to not commit a path to the test bucket, assuming that's what this is.

Reply:
I will add a TODO to make sure I switch over to cloudberry after initial testing.